app/routes/signin-oidc.js

```diff
@@ -180,14 +181,16 @@ export const signinRouteHandlers = [{
               ruralPaymentsAgency: config.ruralPaymentsAgency
             }).code(400).takeover()
         }
-        await raiseIneligibilityEvent(
+
+        const event = getIneligibilityEvent(
           request.yar.id,
           organisation?.sbi,
           crn,
-          organisation?.email,
```

Comment:
Your new method signature is expecting email address to be passed in isn't it, yet it's removed here. not sure if that will line up?

`export const getIneligibilityEvent = (sessionId, sbi, crn, email, exception, reference) =>`

Reply:
good spot, restored.